packages/components/modals/SendModal.tsx

```diff
@@ -0,0 +1,211 @@
+import { Decimal } from "@cosmjs/math";
+import { isDeliverTxFailure } from "@cosmjs/stargate";
+import React, { useCallback } from "react";
+import { useForm } from "react-hook-form";
+import { StyleSheet, TouchableOpacity } from "react-native";
+
+import contactsSVG from "../../../assets/icons/contacts.svg";
+import { useFeedbacks } from "../../context/FeedbacksProvider";
+import { useBalances } from "../../hooks/useBalances";
+import useSelectedWallet from "../../hooks/useSelectedWallet";
+import {
+  getKeplrSigningStargateClient,
+  keplrCurrencyFromNativeCurrencyInfo,
+  NativeCurrencyInfo,
+} from "../../networks";
+import { TransactionForm } from "../../screens/WalletManager/types";
+import {
+  neutral22,
+  neutral33,
+  neutral77,
+  primaryColor,
+} from "../../utils/style/colors";
+import { fontSemibold13 } from "../../utils/style/fonts";
+import { layout } from "../../utils/style/layout";
+import { BrandText } from "../BrandText";
+import FlexCol from "../FlexCol";
+import FlexRow from "../FlexRow";
+import { NetworkIcon } from "../NetworkIcon";
+import { SVG } from "../SVG";
+import { MaxButton } from "../buttons/MaxButton";
+import { PrimaryButton } from "../buttons/PrimaryButton";
+import { TextInputCustom } from "../inputs/TextInputCustom";
+import { SpacerColumn, SpacerRow } from "../spacer";
+import ModalBase from "./ModalBase";
+
+type SendModalProps = {
+  isVisible: boolean;
+  nativeCurrency?: NativeCurrencyInfo;
+  networkId: string;
+  onClose: () => void;
+};
+
+//TODO: Make a reusable component for that, or use an existing one (Which one ? Which other usage for ContactButton ?)
+const ContactButton: React.FC<{ onPress: () => void }> = ({ onPress }) => {
+  //TODO: Remove disabled when contacts are handled
+  return (
+    <TouchableOpacity style={styles.contactsButton} onPress={onPress} disabled>
+      <SVG source={contactsSVG} width={16} height={16} />
```

Comment:
```layout.padding_x2```

Reply:
Sorry, not for `height` and `width`.

`layout.padding_XXX` values could be modified if we want to get more thin or tall spaces between components. But it doesn't must affect the static sizes.